script.js

```diff
@@ -344,7 +398,25 @@ function drawPaddle(ctx, paddle) {
 
 /**
  * @param {CanvasRenderingContext2D} ctx
- * @param {{x: number, y: number}} ball
+ * @param {Score} score
+ * @returns {void}
+ */
+function drawScore(ctx, score) {
+  const fontSize = Math.round(ctx.canvas.height / 4);
+  const center = ctx.canvas.width / 2;
+
+  ctx.font = `bold ${fontSize}px sans-serif`;
+  ctx.fillStyle = "lightgray";
+  ctx.textAlign = "center";
+  ctx.textBaseline = "bottom";
+
+  ctx.fillText(`${score[0]}`, center, SCORE_MARGIN + fontSize + 20);
+  ctx.fillText(`${score[1]}`, center, ctx.canvas.height - SCORE_MARGIN);
+}
```

Comment:
@K0rston Notice the difference on how we draw the score here compared to in class.